src/presenter/logUserPresenter.js

```diff
@@ -0,0 +1,33 @@
+import userModel from "../model/userModel";
+
+class LogUserPresenter {
+
+    onLogIn(){
+        var successfulLogIn = false;
+        for(var iterator = 0; iterator < userModel.state.users.length; ++iterator)
```

Comment:
I would move this inside the model as it is more "business" logic (and not necessarily presentational).